MdeModulePkg/Universal/Console/ConPlatformDxe/ConPlatform.c

```diff
@@ -808,10 +808,14 @@ MatchUsbClass (
   DeviceClass    = DevDesc.DeviceClass;
   DeviceSubClass = DevDesc.DeviceSubClass;
   DeviceProtocol = DevDesc.DeviceProtocol;
-  if (DeviceClass == 0) {
+
+  if ((DeviceClass == 0) ||
+      ((DeviceClass == 0xEF) && (DeviceSubClass == 0x02) && (DeviceProtocol == 0x01)))

```

Comment:
Could the hard code value be replaced with MACRO C name?

Reply:
Yes, this is a good suggestion.

I noticed that there is no IndustryStandard usb file that includes the enumerated usb class/subclass/protocols. (https://www.usb.org/defined-class-codes) It seems like multiple files throughout the usb stack just create their own versions of the defines. Is that intentional?

Reply:
Updated in the latest push to define locally, following the local class/subclass/protocol definitions found in:

https://github.com/tianocore/edk2/blob/master/MdeModulePkg/Bus/Usb/UsbKbDxe/EfiKey.h#L45
https://github.com/tianocore/edk2/blob/master/MdeModulePkg/Bus/Usb/UsbMouseDxe/UsbMouse.h#L29
https://github.com/tianocore/edk2/blob/master/MdeModulePkg/Bus/Usb/UsbMouseAbsolutePointerDxe/UsbMouseAbsolutePointer.h#L29